GLideN64/src/Graphics/OpenGLContext/mupen64plus/mupen64plus_DisplayWindow.cpp

```diff
@@ -71,10 +71,12 @@ bool DisplayWindowMupen64plus::_start()
 	m_screenHeight = get_retro_screen_height();
 	_getDisplaySize();
 	_setBufferSize();
-	
-	LOG(LOG_VERBOSE, "[gles2GlideN64]: Create setting videomode %dx%d", m_screenWidth, m_screenHeight);
 
+#ifdef EGL
+	eglInitialize(eglGetDisplay(EGL_DEFAULT_DISPLAY), nullptr, nullptr);
+#endif
 
+	LOG(LOG_VERBOSE, "[gles2GlideN64]: Create setting videomode %dx%d", m_screenWidth, m_screenHeight);
```

Comment:
Can u change it to "[GlideN64]", I want to use that going forward

Reply:
Done. Changed it in all `LOG(...)` calls in the file.